Resources/Prototypes/Entities/Tiles/bananium.yml

```diff
@@ -0,0 +1,73 @@
+- type: entity
+  id: FloorBananiumEntity
+  name: bananium floor
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Clickable
+  - type: Sprite
+    netsync: false
+    sprite: Tiles/Misc/bananium.rsi
+    drawdepth: FloorTiles
+    layers:
+      - state: bananium
+  - type: Icon
+    sprite: Tiles/Misc/bananium.rsi
+    state: bananium
+  - type: Transform
+    anchored: true
+  - type: FootstepModifier
+    footstepSoundCollection:
+      collection: FootstepClown
+  - type: Tag
+    tags:
+      - Catwalk
+  - type: Construction
+    graph: FloorBananium
+    node: BananiumFloor
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Metallic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 500
+      behaviors:
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+    - trigger:
+        !type:DamageTrigger
+        damage: 200
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          MaterialBananium:
+            min: 0
+            max: 1
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+  - type: Slippery
+    paralyzeTime: 2
+    launchForwardsMultiplier: 1.5
+  - type: StepTrigger
+    intersectRatio: 0.2
+  - type: CollisionWake
+    enabled: false
+  - type: Physics
+    bodyType: Dynamic
```

Comment:
This should be static because it's anchored.

Reply:
it is now static.